util/src/main/java/com/liferay/faces/util/application/ResourceDependencyHandlerFactory.java

```diff
@@ -19,30 +19,17 @@
 
 
 /**
- * @author  Neil Griffin
+ * This class provides the contract for a factory which can create {@link ResourceDependencyHandler}s.
+ *
+ * @author  Kyle Stiemann
  */
-public abstract class ComponentResourceWrapper implements ComponentResource, FacesWrapper<ComponentResource> {
-
-	@Override
-	public boolean isRenderable() {
-		return getWrapped().isRenderable();
-	}
-
-	@Override
-	public String getId() {
-		return getWrapped().getId();
-	}
-
-	@Override
-	public String getLibrary() {
-		return getWrapped().getLibrary();
-	}
-
-	@Override
-	public String getName() {
-		return getWrapped().getName();
-	}
-
-	@Override
-	public abstract ComponentResource getWrapped();
+public abstract class ResourceDependencyHandlerFactory implements FacesWrapper<ResourceDependencyHandlerFactory> {
```

Comment:
Github is too smart for its own good and it thinks this diff is a rename or move. This is an API change: added `ResourceDependencyHandlerFactory`.